Packages/com.unity.inputsystem/InputSystem/NativeInputRuntime.cs

```diff
@@ -393,8 +393,7 @@ public Action onProjectChange
 
         public void SendAnalytic(InputAnalytics.IInputAnalytic analytic)
         {
-        #if ENABLE_CLOUD_SERVICES_ANALYTICS
-            #if (UNITY_EDITOR)
+            #if (UNITY_EDITOR && ENABLE_CLOUD_SERVICES_ANALYTICS)
```

Comment:
I'm not 100% sure on this

This documentation about  Unity**Engine**.Analytics
https://docs.unity3d.com/ScriptReference/Analytics.Analytics.html 
Say to use ENABLE_CLOUD_SERVICES_ANALYTICS

But the  **Editor**Analytics API doesn't mention it
https://docs.unity3d.com/ScriptReference/EditorAnalytics.html
https://docs.unity3d.com/ScriptReference/EditorAnalytics.RegisterEventWithLimit.html

https://docs.unity3d.com/ScriptReference/EditorAnalytics.html

Reply:
I think this might be around the wrong section. 

I.e. needed for this part #elif (UNITY_ANALYTICS) // Implicitly: !UNITY_EDITOR

Reply:
See also https://jira.unity3d.com/browse/ISX-2092

Reply:
Have you tested this for a console platform @ekcoh ? 
I believe I tried this logic in the past and it didn't work. Because if you're in the Editor and the platform cannot compile Analytics ( ENABLE_CLOUD_SERVICES_ANALYTICS is false), it would still try to call `EditorAnalytics.SendAnalytic(analytic);` and error on Console.

Reply:
Sorry that I did not respond to this promptly, the current version of this I tested on one console platform in editor and player builds.